src/select/Select.ts

```diff
@@ -250,7 +250,7 @@ export default class Select extends SelectBase<SelectProperties> {
 		});
 	}
 
-	renderNativeSelect(): DNode {
+	protected renderNativeSelect(): DNode {
```

Comment:
I'm all for this change, but I'm not sure we should touch visibility as part of this theme-specific PR.